Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/NSAttributedString+MessageFormatting.swift

```diff
@@ -25,6 +25,10 @@ import WireDataModel
 
 extension NSAttributedString {
     
+    struct Wire {
+        static var defaultParagraphSpacing: CGFloat = 8
+    }
```

Comment:
We should use `enum` here since we never create a Wire struct instance.

```suggestion
extension CGFloat {
    enum DefaultParagraph {
        static let spacing: CGFloat = 8
    }

```

Reply:
and this can be put outside extension NSAttributedString

Reply:
👍 I'll find a better place for it.

Reply:
Moved it to `CGFloat+Const.swift`